examples/cryptocurrency-advanced/tutorial/frontend.md

```diff
@@ -1,320 +0,0 @@
-# Cryptocurrency frontend tutorial
```

Comment:
What will happen to this guide in the future?

Reply:
This guide duplicates [the one on exonum.com](https://exonum.com/doc/version/1.0.0-rc.1/get-started/light-client/), thus I think it can be safely removed.